sdk/identity/identity/samples/AzureIdentityExamples.md

```diff
@@ -603,12 +603,34 @@ function withSystemAssignedManagedIdentityCredential() {
 
   const client = new SecretClient("https://key-vault-name.vault.azure.net", credential);
 }
+```
+
```

Comment:
To be totally honest, I feel like all this is doing is duplicating the types that we already provide. I wanted to offer more, like a "for more information on user-assigned managed identities please refer to [the documentation]()"

But, the only documentation I was able to find on this was https://learn.microsoft.com/en-us/azure/app-service/overview-managed-identity?tabs=portal%2Chttp#rest-endpoint-reference and I don't think it's very clear either. Anyone have a link that I could share? Or is this sufficient?

Reply:
Thanks!